### ISSUE TITLE
How is this working as intended?

### ISSUE THREAD

**Issue by dustybutcheeks** [ValveSoftware/Dota2-Gameplay#14910]
### Description

How am I supposed to gain behviour score if you get 5 reports for losing mid and a low priority sentence for it. What the hell are you smoking? How many complaints do you have to receive that this shit is broken. You banned the only account I can play with friends on that wasn't even a smurf because it had the same mmr as my main you are doing your job so badly, Revert these shit changed and then don't touch the game again.

### Example Match ID (and possibly Timestamp)

7531433122

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/144056214/2ac06177-31b9-40e2-ac29-06711d732c44)

**Comment by dustybutcheeks** [comment]
Actually just perma ban my main so I can make a new one like Mason its better than putting up with this bs.

**Comment by Questy33** [comment]
Explain what happened, bro? Do you receive reports for a lost line? But you don’t break items, you don’t ruin them and you don’t cost AFK?

**Comment by dustybutcheeks** [comment]
> Explain what happened, bro? Do you receive reports for a line loose? But you don’t break items, you don’t ruin them and you don’t cost AFK?

Yes. I played the game out to my bestest and said nothing and Iget low priority and -400 behaviour score the system is outrageously broken.
